src/compiler/diagnosticMessages.json

```
@@ -8433,5 +8433,21 @@
     "String literal import and export names are not supported when the '--module' flag is set to 'es2015' or 'es2020'.": {
         "category": "Error",
         "code": 18057
+    },
+    "Default imports aren't allowed for deferred imports.": {
+        "category": "Error",
+        "code": 18058
+    },
+    "Named imports aren't allowed for deferred imports.": {
+        "category": "Error",
+        "code": 18059
+    },

```

We mostly use "are not" (38 uses) over "aren't" (1 use) in diagnostic messages, much like we prefer "is not" (99 uses) over "isn't" (1 use).

```suggestion
    "Default imports are not allowed in a deferred import.": {
        "category": "Error",
        "code": 18058
    },
    "Named imports are not allowed in a deferred import.": {
        "category": "Error",
        "code": 18059
    },
```